WeartGloves/CMakeLists.txt

```diff
@@ -0,0 +1,100 @@
+#/*
+# Copyright 2021 Istituto Italiano di Tecnologia (IIT)
+# @author Ehsan Ranjbari <ehsan.ranjbari@iit.it>
+#* CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+#*/
+cmake_minimum_required(VERSION 3.5)
+
+project(YARPWeArtGlove VERSION "0.0.1"
+                  LANGUAGES CXX C)
+
+set(WeArtGloveLibName "WeArtGloveLib")
+set(WeArtGloveDevName "WeArtGlove")
```

Comment:
Personal opinion, no need to change anything. I am not a big fan of the undirection, personally I found the code more readable if we use directly `WeArtGloveLib` and `WeArtGlove` without any additional variabale.